pkg/operator/starter.go

```diff
@@ -71,10 +71,13 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 	versionRecorder := status.NewVersionGetter()
 	versionRecorder.SetVersion("operator", status.VersionForOperatorFromEnv())
 
+	operatorclient.OperatorNamespace = cc.OperatorNamespace
+
 	kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient,
```

Comment:
Let's check library-go implementation for the Static Resources Controller (especially informers),
my guess is that there is no informer/client for NetworkPolicy yet.